com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkPrefab.cs

```diff
@@ -4,6 +4,13 @@
 
 namespace MLAPI.Configuration
 {
+    public enum NetworkPrefabOverride
+    {
+        Unset,
```

Comment:
Nit: I think I would pick 'None' else at first glance sounds like 'Unset' is a 'needs-to-be-initialized' state

Reply:
Yes, I agree.  None is what that should be.

Reply:
Actually, I think this should be "Default".